docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
       dockerfile: Dockerfile
     volumes:
       - .:/opt/app
-    tmpfs: /opt/app/inbox_sync.egg-info
+    user: root
```

Comment:
Now with coverage writing to different locations it is too annoying to maintain those path mappings. Also tmpfs does not work consistently between Docker Desktop and Docker CE.

Reply:
Note that the image we push does not run as root as shipped.